### PR TITLE
Added endless Wondertrade and cleanend my useless stuff.

### PR DESCRIPTION
**Pull request by realTobby**
I added the endless function for Wondertrading.
You basically can just let the Wondertrade go on, while you are not on the 3ds or Computer.
No need to get notified by the BOT that a session of trades is done.

Code is not really beautiful but works for me.
Hope this fits into this program.

### CODE REVIEW

File: PKMN-NTR/MainForm.cs [hunk 1]
```diff
@@ -3394,8 +3397,85 @@ private void srFilterLoad_Click(object sender, EventArgs e)
...
+        // when the start button is pressed the endless wondertrade will start
+        private async void buttonWTStartEndless_Click(object sender, EventArgs e)
+        {
+            DialogResult dialogResult = MessageBox.Show("This scirpt will try to Wonder Trade " + WTtradesNo.Value + " pokémon, starting from the slot " + WTSlot.Value + " of box " + WTBox.Value + ". CAUTION: THIS BOT WILL TRADE ENDLESSLY! WHEN THE BOT REACHES 0 IT RETURNS TO THE VALUES YOU TYPED IN! ALL POKEMON WILL BE WONDERTRADED AGAIN", "Wonder Trade Bot", MessageBoxButtons.OKCancel, MessageBoxIcon.Warning);
+            if (dialogResult == DialogResult.Cancel && WTtradesNo.Value > 0)
+            {
+                return;
+            }
+
+            // get standard value
```
Comment (jackmax) [inline PKMN-NTR/MainForm.cs:3529]: I'll change this to "initial value", "standard value" sounds weird.

File: PKMN-NTR/MainForm.cs [hunk 2]
```diff
@@ -310,7 +313,7 @@ public MainForm()
...
..., evDEFNum, evSPANum, evSPDNum, evSPENum, HypT_HP, HypT_Atk, HypT_Def, HypT_SpA, HypT_SpD, HypT_Spe, move1, move2, move3, move4, relearnmove1, relearnmove2, relearnmove3, relearnmove4, otName, dTIDNum, dSIDNum, pkLang, showItems, showMedicine, showTMs, showBerries, showKeys, itemWrite, ReloadFields, playerName, pokeName, TIDNum, pokeTID, SIDNum, pokeSID, moneyNum, pokeMoney, milesNum, pokeMiles, totalFCNum, pokeTotalFC, bpNum, pokeBP, Lang, pokeLang, hourNum, minNum, secNum, pokeTime, cloneBoxTo, cloneSlotTo, cloneCopiesNo, cloneBoxFrom, cloneSlotFrom, cloneDoIt, writeBoxTo, writeSlotTo, writeCopiesNo, writeAutoInc, writeBrowse, writeDoIt, deleteBox, deleteSlot, deleteAmount, deleteKeepBackup, delPkm, manualDUp, ManualDDown, manualDLeft, manualDRight, manualA, manualB, manualX, manualY, manualL, manualR, manualStart, manualSelect, touchX, touchY, manualTouch, StickY, StickX, StickNumY, StickNumX, StickSend, manualSR, modeBreed, boxBreed, eggsNoBreed, bFilterLoad, filterBreeding, ESVlistSave, TSVlistNum, TSVlistAdd, TSVlistRemove, TSVlistSave, TSVlistLoad, readESV, runBreedingBot, typeLSR, srFilterLoad, filtersSoftReset, RunLSRbot, WTBox, WTSlot, WTtradesNo, RunWTbot, WTcollectFC };
```
Comment (jackmax) [inline PKMN-NTR/MainForm.cs]: Does this bot work for Gen 7? If it does, I'll add that button here.

**Comment by jackmax** [comment]
Does this bot work for Gen 7? If it does, I'll enable the button when connecting.

**Comment by realTobby** [comment]
I didnt tested it for Gen 7, but I think it should. I will test this now.

2016-12-27 14:16 GMT+01:00 jackmax <notifications@github.com>:

**Comment by realTobby** [comment]
Iam currently unable to test this sadly. My Pokemon Moon is is keep giving
me an error message when I try to connect to the internet ._.
When needed I can remove the Gen7 parts of this bot (then it would be ORAS
exclusive >_>)

I cant seem to get it to work with Gen7 for me.

2016-12-28 8:00 GMT+01:00 Tobias Kattanek <tkattanek00@gmail.com>:

**Comment by realTobby** [comment]
Okay...I need to apologize.

The bot is working just fine with Gen 7 and all Gen 6 games there was a
mistake on my end.
I had an NTR plugin for my Pokemon Moon...that was the problem why I
couldnt connect to the Internet.
Now I removed these plugins and the EndlessBot works here too :)

So you can enable the Button for Gen 7 too ^^

2016-12-28 8:21 GMT+01:00 Tobias Kattanek <tkattanek00@gmail.com>:

**Pull request merged**